index.d.ts

```diff
@@ -1,3 +1,14 @@
+declare namespace decamelize {
+	interface Options {
+		/**
+		Preserve blocks of uppercase characters: `fooBARAction` → `foo_BAR_action`.
+
```

Comment:
Can you add an example code block here instead and show an example with and without the option as `true`?